docs/en/enterprise-edition/content-collections/governance/custom-build-policies/custom-build-policies.adoc

```diff
@@ -15,9 +15,9 @@ An *AI & Machine Learning* category enables granular control over build configur
 In addition, Prisma Cloud also supports identification of custom secrets that you can define using regular expression patterns.
 While defining the regular expression patterns consider the following parameters:
 
-* You must ensure the `RegEx` pattern meets standard regular expression formatting standards.
+* You must ensure the `RegEx` pattern meets standard regular expression formatting standards for Python based regular expressions.
```

Comment:
Ensure that the RegEx pattern complies with Python's regular expression syntax standards.